styles/length_finder.h

```diff
@@ -34,7 +34,7 @@ class LengthFinder {
     if (say_it_ && millis() - last_speak_ > 3000) {
       last_speak_ = millis();
       say_it_ = false;
-#ifdef ENABLE_AUDIO      
+#if defined(ENABLE_AUDIO) && !defined(DISABLE_TALKIE)
```

Comment:
This should be an error.
Lengthinder is useless without talkie.

Reply:
thought user could just use Serial Monitor in this case?